app/api/schema/settings.py

```diff
@@ -27,6 +27,9 @@ class Meta:
     # Tagline for the application. (Eg. Event Management and Ticketing, Home)
     tagline = fields.Str(allow_none=True)
 
+    # Order Expiry Time
+    order_expiry_time = fields.Integer(allow_none=True, default=15, validate=lambda n: 1 <= n <= 60)
```

Comment:
In the model you've specified `nullable=False`. Please change this.

Reply:
I think there is no need to change the `allow_none`, There was a need to make a change in `models` which I have made

Reply:
@kushthedude You need to change this as well. It's basically like restricting models to store a null value and allowing schema to pass a non null value. There should be a consistency between the two. Hence please make `allow_none=False`